app/main.py

```diff
@@ -16,4 +16,12 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {"immutable": [lucky_number,
+                                  pi,
+                                  name,
+                                  profile_info,
+                                  one_is_a_prime_number],
+                    "mutable": [marks,
+                                my_favourite_films,
+                                collection_of_coins
+                                ]}
```

Comment:
```suggestion
sorted_variables = {
    "immutable": [
        lucky_number,
        pi,
        name,
        profile_info,
        one_is_a_prime_number,
    ],
    "mutable": [
        marks,
        my_favourite_films,
        collection_of_coins,
    ],
}

```

Reply:
In general, was the solution correct, except for syntax errors? Did I understand correctly?